packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -368,14 +368,12 @@
                         data-term="{{:: 'hosting_dashboard_service_cdn' | translate }}"
                     >
                         <oui-tile-description>
-                            <span
-                                data-ng-if=""
-                            >
+                            <span data-ng-if="">
```

Comment:
**nitpick(non-blocking):** Following expression will always be evaluated as falsy, so we can safely get rid of this `<span>` element.

Reply:
Looks like its the old, existing code.

Reply:
Is existing code, not add by me. Lint has modified structure. This span has the content. It's a temporary deletion? Should I delete it?

Reply:
Yes please! We can safely remove this `<span>` element and its translation here:
https://github.com/ovh/manager/blob/5251e0ec44723fce706bd5283d759de0c59a5899/packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json#L151

Reply:
In all cases, as @antleblanc mentioned it, the `ng-if` will always returns false => the chlid span has its own condition, so yes you can remove it